test/data/Mapping.de-de.txt

```diff
@@ -0,0 +1,7 @@
+Wirtschaft	0
```

Comment:
This is associated with `lm.sample.txt`, I guess? So I wonder if `lm.labels.txt` might be a better name, just to make clear the two files are in some way associated -- `Mapping.de-de.txt` is not entirely clear.